### DISCUSSION TITLE
Loading FutureProviders in Isolate

### DISCUSSION THREAD

**Discussion by muhammadsqln** [rrousselGit/riverpod#1299] [Q&A]
We have an app for managing a crew's timesheet where we allocate each crew member's time to activities that they worked on a day to say basis. 
All the data I pull for each day is coming from a FutureProvider. 

This data is large and so I would like to load this data before even getting to the screen so that it's readily available once the user lands there. I want to load it (and other FutureProviders) in the background using an Isolate. How would I go about doing this?

**Comment by rrousselGit** [comment]
You can use Flutter's compute function inside a FutureProvider.
